translations/README.md

```diff
@@ -1,4 +1,4 @@
 # Translation Resources
-Translations are managed through [Lokalise](https://lokalise.co/). If you'd like to contribute, you can [join the project here](https://lokalise.co/signup/3420425759f6d6d241f598.13594006/all/). For more details, see our [translation guidelines documentation](https://developers.home-assistant.io/docs/en/internationalization_translation.html).
+Translations are managed through [Lokalise](https://lokalise.com/). If you'd like to contribute, you can [join the project here](https://lokalise.co/signup/3420425759f6d6d241f598.13594006/all/). For more details, see our [translation guidelines documentation](https://developers.home-assistant.io/docs/en/internationalization_translation.html).
```

Comment:
Lets just point to our developer documentation instead, as all links and details there are kept up to date.

Reply:
> Lets just point to our developer documentation instead, as all links and details there are kept up to date.

It's now updated to the developer docs

Reply:
the last URL needs an update as well

Reply:
> the last URL needs an update as well
The last URL is fixed too now.